kids-fly/src/index.js

```diff
@@ -1,17 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
-import { BrowserRouter as Router} from "react-router-dom";
+import LandingPage from './components/LandingPage';
+import Header from './components/Header';
 
-ReactDOM.render(
-  <Router>
-    <App />
-  </Router>, 
-  document.getElementById('root'));
```

Comment:
lines 8 thru 12 do the same thing lines 17 thru 23 do

Reply:
Got it! I did resolve some of the merge conflicts..